config.yaml

```diff
@@ -1,18 +1,32 @@
 options:
-  prometheus-image-path:
+  image-tags-url:
     type: string
     description: |
-      The location of the image to use,
-      e.g. "registry.example.com/prometheus:v1".
-
-      This setting is required.
-    default: "prom/prometheus:latest"
+      URL at which all the tags for the Prometheus container images can
+      be viewed and verified.
+    default: "https://registry.hub.docker.com/v1/repositories/prom/prometheus/tags"
+  image-registry-path:
+    type: string
+    description: |
+      Path from which to pull the Prometheus container image.
+    default: "prom/prometheus"
```

Comment:
Doesn't registry-path overlap with image-tags-url?

Reply:
I'm not sure on this one but I don't think there's much overlap. I'll have to do some more digging.

Reply:
The goal here is to completely remove references to registry path, not split it into multiple configs. At this time, there's no reason to allow for a consumer to override the image location. Eventually, we'll be leveraging resources as the correct path to override. Let's not add config items which will be removed in the near future.